src/widget.js

```diff
@@ -56,18 +56,23 @@ Widget.prototype = {
         this.setState(this.state);
         break;
       case 'sync-req-done':
+        this.syncInProgress = true;
```

Comment:
By the way, I opened https://github.com/remotestorage/remotestorage.js/issues/1173 to improve this considerably.